arcade/camera/camera_2d.py

```diff
@@ -165,26 +101,75 @@ def from_raw_data(
             )
 
         _pos = position or (half_width, half_height)
-        _data = CameraData(
+        self._camera_data = CameraData(
             position=(_pos[0], _pos[1], 0.0),
             up=(up[0], up[1], 0.0),
             forward=(0.0, 0.0, -1.0),
             zoom=zoom
         )
-
-        _projection: OrthographicProjectionData = OrthographicProjectionData(
+        self._projection_data: OrthographicProjectionData = OrthographicProjectionData(
             left=left, right=right,
             top=top, bottom=bottom,
             near=near, far=far,
             viewport=viewport or (0, 0, width, height)
         )
+        self._ortho_projector: OrthographicProjector = OrthographicProjector(
+            window=self._window,
+            view=self._camera_data,
+            projection=self._projection_data
+        )
+
+    @classmethod
+    def from_camera_data(cls, *,
+                         camera_data: Optional[CameraData] = None,
+                         projection_data: Optional[OrthographicProjectionData] = None,
+                         render_target: Optional[Framebuffer] = None,
+                         window: Optional["Window"] = None) -> Self:
+        """
+        Create a Camera2D from provided CameraData or/and OrthographicProjectionData objects.
```

Comment:
TL;DR: Something like this for explaining the purpose of the function?
```suggestion
        """Make a ``Camera2D`` directly from data objects.

        This :py:class:`classmethod` allows advanced users to:

        #. skip or replace the default validation
        #. share ``camera_data`` or ``projection_data`` between cameras

       .. warning:: Be careful when sharing data objects!

                    **Any** action on a camera which changes a shared
                    object changes it for **every** camera which uses
                    the same object.

       .. list-table::
          :header-rows: 1

          * - Shared Value
            - Example Use(s)

          * - ``camera_data``
            - Mini-maps, reflection, and ghosting effects.

          * - ``projection_data``
            - Simplified rendering configuration

          * - ``render_target``
            - Complex rendering setups

```

Reply:
I don't understand the dots before the warning or the list-table, and also the alignment of those docstrings.
I'll submit the change anyway in a few moments

Reply:
@pushfoo yeah that looks good to me!